src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -349,65 +369,76 @@ export default class ReviewConfirmDissolution extends Mixins(DateMixin) {
     })
   }
 
-  private getHeaderNumber (sectionName): number {
-    const userHeaderNumbers: any = { documentDelivery: 1, certify: 2 }
-    const premiumHeaderNumbers: any = { documentDelivery: 1, folioNumber: 2, certify: 3 }
-    const staffHeaderNumbers: any = { documentDelivery: 1, certify: 2, courtOrder: 3, staffPayment: 4 }
-
-    let headerNumbers = userHeaderNumbers
-    if (this.isPremiumAccount) {
-      headerNumbers = premiumHeaderNumbers
-    } else if (this.isRoleStaff) {
-      headerNumbers = staffHeaderNumbers
+  /** Returns the header number for the specified section name. */
+  getHeaderNumber (sectionName: string): number {
+    const userHeaderNumbers: any = {
+      documentDelivery: 1,
+      certify: 2
+    }
+    const premiumHeaderNumbers: any = {
+      documentDelivery: 1,
+      folioNumber: 2,
+      certify: 3
     }
-    return headerNumbers[sectionName]
+    const staffHeaderNumbers: any = {
+      documentDelivery: 1,
+      certify: 2,
+      courtOrder: 3,
+      staffPayment: 4
+    }
+
+    if (this.isPremiumAccount) return premiumHeaderNumbers[sectionName]
+    if (this.isRoleStaff) return staffHeaderNumbers[sectionName]
+    return userHeaderNumbers[sectionName]
   }
 
   /** Is true when the Define Dissolution conditions are not met. */
-  private get isDefineDissolutionInvalid (): boolean {
```

Comment:
I removed the visibility because VS Code shows these as not used (because they are in the template not the script). And also, these are not really private -- something outside this class could look at this without concern.